@Test public void appAndTestsInOneFile() throws Exception {
assertThat(output,containsString("OK (1 test)"));
}